src/bridge.js

```diff
@@ -444,16 +445,10 @@ Bridge.prototype._customiseAppserviceThirdPartyLookup = function(lookupControlle
 
         this.addAppServicePath({
             method: "GET",
-            path: "/_matrix/app/:ver/thirdparty/protocol/:protocol",
+            path: "/_matrix/app/:version/thirdparty/protocol/:protocol",
```

Comment:
Why does this one not need the `access_token` to be deleted like most other AppServicePaths?
Should the `delete req.query.access_token;` be moved to a more central function?

Reply:
Because it's not using req.query in the function below. We only want to remove it for the two functions which use the whole of req.query.

The reason we use the whole of `req.query` is because of this unfortunate API shape: https://matrix.org/docs/spec/application_service/r0.1.2#get-matrix-app-v1-thirdparty-user-protocol